src/components/common/Extensions/UserNavigation/UserNavigation.tsx

```diff
@@ -103,46 +102,32 @@ const UserNavigation: FC = () => {
       )}
       {isButtonVisible && !isWalletConnected && (
         <Button
-          mode="quinary"
+          mode="tertiary"
           isFullRounded
           setTriggerRef={setWalletTriggerRef}
           onClick={connectWallet}
-          className={clsx('md:border-gray-200 md:hover:border-blue-400', {
-            'px-4 py-2.5 border-base-white text-gray-400':
-              isWalletVisible && isMobile,
-            'p-0': !isWalletVisible && isMobile,
+          iconName={isWalletVisible && isMobile ? 'close' : 'cardholder'}
+          className={clsx({
+            '!border-base-white': isWalletVisible && isMobile,
```

Comment:
'!border-base-white' is it correct?

Reply:
Yes, we can't override this style in tailwind in different way in this case :/